dts/arm/nxp/nxp_ke1xz.dtsi

```diff
@@ -167,17 +167,18 @@
 			clocks = <&pcc 0x134 KINETIS_PCC_SRC_NONE_OR_EXT>;
 		};
 
-		gpios0: gpios0 {
+		gpios0: gpios0@400ff000 {
 			compatible = "nxp,gpio-cluster";
 			interrupts = <7 2>;
-
+			reg = <0x400ff000 0x200>;
+			ranges = <0x0 0x400ff000 0x200>;
 			#address-cells = <1>;
 			#size-cells = <1>;
 
 			gpioa: gpio@400ff000 {
 				compatible = "nxp,kinetis-gpio";
 				status = "disabled";
-				reg = <0x400ff000 0x40>;
+				reg = <0x0 0x40>;
```

Comment:
Should we also update the register address for `gpioe`?

Reply:
hm I thought I did woops

clearly the answer is yes I don't know why you would phrase it like a question